modules/core/shared/src/main/scala/gem/enum/GnirsDecker.scala

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.syntax.eq._
+import cats.instances.string._
+import gem.util.Enumerated
+
+/**
+ * Enumerated type for GNRIS Decker.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GnirsDecker(
+  val tag: String,
+  val shortName: String,
+  val longName: String,
+  val obsolete: Boolean
+) extends Product with Serializable
+
+object GnirsDecker {
+
+  /** @group Constructors */ case object Acquisition extends GnirsDecker("Acquisition", "ACQ", "Acquisition", false)
+  /** @group Constructors */ case object PupilViewer extends GnirsDecker("PupilViewer", "PV", "Pupil viewer", false)
+  /** @group Constructors */ case object ShortCamLongSlit extends GnirsDecker("ShortCamLongSlit", "SHORT_CAM_LONG_SLIT", "Short camera long slit", false)
+  /** @group Constructors */ case object ShortCamCrossDispersed extends GnirsDecker("ShortCamCrossDispersed", "SHORT_CAM_XD", "Short camera cross dispersed", false)
+  /** @group Constructors */ case object Ifu extends GnirsDecker("Ifu", "IFU", "IFU", true)
+  /** @group Constructors */ case object LongCamLongSlit extends GnirsDecker("LongCamLongSlit", "LONG_CAM_LONG_SLIT", "Long camera long slit", false)
+  /** @group Constructors */ case object Wollaston extends GnirsDecker("Wollaston", "WOLLASTON", "Wollaston", true)
```

Comment:
Why is the Wollaston in the GnirsDecker?  (it's really in the prism turret)

Reply:
It was there in the original OCS code, but it felt odd to me too. I'll remove it.

Reply:
Sorry, I understand now.  The decker has a position for the Wollaston prism, and this must set that position.

Reply:
Here are the Decker positions from the GNIRS control software:
![spectacle w26012](https://user-images.githubusercontent.com/11798572/33746863-dbd5dabe-db63-11e7-8ceb-8c0ecf52fed1.png)